backend/foodbank_southlondon/api/views.py

```diff
@@ -23,5 +23,5 @@ class Calendars(flask_restx.Resource):
     @rest.marshal_with(_calendar_model)
     def get(self):
         return {
-            "calendar_ids": [parse.quote(calendar["calendar_id"]) for calendar in flask.current_app.config[_FBSL_COLLECTION_CENTRES].values()],
+            "calendar_ids": [calendar["calendar_id"] for calendar in flask.current_app.config[_FBSL_COLLECTION_CENTRES].values()],
```

Comment:
We didn't need to quote these as they are marshalled into a JSON string automatically

Reply:
Oh, cool. In that case, the import can be dropped too. I'll make a commit.